src/HUD.h

```diff
@@ -0,0 +1,162 @@
+#ifndef HUD_H
+#define HUD_H
+
+#include <osg/Camera>
+#include <osg/Geode>
+#include <osg/Texture2D>
+#include <osgDB/ReadFile>
+#include <osg/Geometry>
+#include <osgText/Text>
+#include <osgText/Font>
+#include <osgViewer/Viewer>
+#include <osgViewer/ViewerEventHandlers>
+#include <sstream>
+#include <osgSim/ShapeAttribute>
+#include <codecvt>
+#include <locale>
+#include <unordered_map>
+
+static osg::ref_ptr<osgText::Text> g_hudText;

```

Comment:
```suggestion
extern osg::ref_ptr<osgText::Text> g_hudText;
extern osg::ref_ptr<osgText::Text> g_hudTextShd;
```
Declaration of static variables in header files is rather bad practice.